demo/openapi.yaml

```diff
@@ -107,6 +107,11 @@ paths:
       summary: Add a new pet to the store
       description: Add new pet to the store inventory.
       operationId: addPet
+      x-badges:
+        - name: MyBadge
+          type: primary
+        - name: Experimental
+          type: experimental
```

Comment:
At first, my idea was to go with `name` and `color` and allow the user to specify either a color name (i.e. `red`, `green`, `blue`, `light-red`, etc) or even a hex color value, but decided to start smaller and support a single new color that follows the same `ColorSettings` format.

New color is `experimental` that shows up a light purple/pinkish color.